libs/SalesforceOAuth/SalesforceOAuthUnitTests/SFOAuthFlowAndDelegate.m

```diff
@@ -101,6 +137,15 @@ - (void)beginTokenEndpointFlow:(SFOAuthTokenEndpointFlow)flowType {
     [self log:SFLogLevelDebug format:@"%@ called.", NSStringFromSelector(_cmd)];
     self.beginTokenEndpointFlowCalled = YES;
     self.tokenEndpointFlowType = flowType;
+    
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(5.0 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```

Comment:
You could replace the `5.0` with `self.timeBeforeTokenRefreshCompletion`, or somesuch.